src/analyses/static_analysis.cpp

```diff
@@ -32,9 +32,9 @@ exprt static_analysis_baset::get_guard(
   next++;
 
   if(next==to)
-    return boolean_negate(from->guard);
+    return boolean_negate(from->get_condition());
 
-  return from->guard;
+  return from->get_condition();
```

Comment:
Could this please get the same changes as `flow_insensitive_abstract_domain_baset::get_guard`? I may have been nicer to break out that sort of improvement into a separate commit, but if we're doing it in one place it should happen everywhere.